src/routes/onboarding/glucose-notification-route.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+const textStyles = {
+  base: "font-text text-section-subtext font-normal leading-[24px] text-center",
+  confirmationTitle:
+    "font-text text-section-header font-medium leading-[38.4px] text-center max-w-[358px] mx-auto",
+};
+
+const GlucoseNotification: React.FC = () => {
+  return (
+    <div className="flex flex-col max-w-screen-md bg-lightPink relative overflow-hidden m-auto pb-80 h-screen">
+      <div className="z-10">
+        <div className="flex flex-col gap-6 items-center pt-32">
+          <h2 className={textStyles.confirmationTitle}>
+            Get notified to measure glucose!
+          </h2>
+
+          <div className="w-[358px] flex flex-col items-center gap-5">
+            <p className={textStyles.base}>
+              We will send notifications to your email with reminders to measure
+              your glucose levels.
+            </p>
```

Comment:
I made a slight edit to this text compared to Figma because of grammar. Curious if this is over-stepping for the client or not.